[BUG] Coroot segfault when receiving logs from a cluster containing a flux instance

I have a multi cluster setup with different clusters forwarding theirs logs and metrics to a central cluster and the coroot pod in the central cluster keeps segfaulting since i added flux in one of the clusters.

```
I0630 11:58:49.351909       1 main.go:46] edition: Community
I0630 11:58:49.352109       1 main.go:47] version: 1.23.0
I0630 11:58:49.352491       1 main.go:63] database type: sqlite
I0630 11:58:49.607116       1 cache.go:136] loaded from disk in 234ms
I0630 11:58:49.607399       1 compaction.go:81] compaction worker started
I0630 11:58:49.958100       1 server.go:63] grpc server: serving on :4317
I0630 11:58:49.958251       1 main.go:265] listening on :8080
I0630 11:58:49.991443       1 constructor.go:175] jh12pq95: got 1 nodes, 33 apps in 38ms
I0630 11:58:49.998203       1 auditor.go:104] jh12pq95: audited 33 apps in 6ms
I0630 11:58:50.031222       1 constructor.go:175] 4fayh2b7: got 1 nodes, 29 apps in 23ms
I0630 11:58:50.035760       1 auditor.go:104] 4fayh2b7: audited 29 apps in 4ms
I0630 11:58:50.055415       1 constructor.go:175] q68h9g3p: got 1 nodes, 32 apps in 18ms
I0630 11:58:50.058998       1 auditor.go:104] q68h9g3p: audited 32 apps in 3ms
E0630 11:58:56.747513       1 logs.go:193] clickhouse tables not ready for project 4fayh2b7
I0630 11:58:59.608495       1 compaction.go:92] compaction iteration started
I0630 11:59:00.574824       1 migration.go:44] clickhouse tables migration done in 10.967s
I0630 11:59:02.272572       1 constructor.go:175] jh12pq95: got 1 nodes, 30 apps in 15ms
I0630 11:59:02.307191       1 constructor.go:175] 4fayh2b7: got 1 nodes, 29 apps in 10ms
I0630 11:59:02.344041       1 constructor.go:175] q68h9g3p: got 1 nodes, 31 apps in 23ms
I0630 11:59:02.772074       1 updater.go:208] 4fayh2b7: cache updated in 12.149s
I0630 11:59:02.800683       1 constructor.go:175] 4fayh2b7: got 1 nodes, 29 apps in 21ms
I0630 11:59:02.801805       1 constructor.go:175] jh12pq95: got 1 nodes, 30 apps in 11ms
I0630 11:59:02.805639       1 auditor.go:104] 4fayh2b7: audited 29 apps in 4ms
I0630 11:59:02.807662       1 deployments.go:38] 4fayh2b7: checked 16 apps in 1ms
I0630 11:59:02.809444       1 incidents.go:130] 4fayh2b7: checked 3 apps in 3ms
I0630 11:59:02.839339       1 constructor.go:175] q68h9g3p: got 1 nodes, 30 apps in 12ms
I0630 11:59:03.104796       1 updater.go:208] jh12pq95: cache updated in 12.487s
I0630 11:59:03.133569       1 updater.go:208] q68h9g3p: cache updated in 12.507s
I0630 11:59:04.159018       1 alerts.go:135] 4fayh2b7: evaluated 289 alerts in 1.353s
I0630 11:59:04.159092       1 watchers.go:202] 4fayh2b7: iteration done in 1.386s
I0630 11:59:04.184206       1 space_manager.go:238] storage manager is disabled for ClickHouse on S3
I0630 11:59:04.229471       1 constructor.go:175] jh12pq95: got 1 nodes, 33 apps in 43ms
I0630 11:59:04.234509       1 auditor.go:104] jh12pq95: audited 33 apps in 4ms
I0630 11:59:04.235868       1 deployments.go:38] jh12pq95: checked 12 apps in 1ms
I0630 11:59:04.241729       1 incidents.go:130] jh12pq95: checked 11 apps in 7ms
I0630 11:59:05.104247       1 alerts.go:135] jh12pq95: evaluated 310 alerts in 869ms
I0630 11:59:05.104432       1 watchers.go:202] jh12pq95: iteration done in 920ms
I0630 11:59:05.147629       1 constructor.go:175] q68h9g3p: got 1 nodes, 32 apps in 42ms
I0630 11:59:05.150938       1 auditor.go:104] q68h9g3p: audited 32 apps in 3ms
I0630 11:59:05.151892       1 deployments.go:38] q68h9g3p: checked 9 apps in 0s
I0630 11:59:05.168647       1 incidents.go:130] q68h9g3p: checked 16 apps in 17ms
I0630 11:59:05.301433       1 alerts.go:135] q68h9g3p: evaluated 301 alerts in 150ms
I0630 11:59:05.301488       1 watchers.go:202] q68h9g3p: iteration done in 197ms
I0630 11:59:09.608039       1 compaction.go:92] compaction iteration started
I0630 11:59:19.607495       1 compaction.go:92] compaction iteration started
I0630 11:59:29.607892       1 compaction.go:92] compaction iteration started
I0630 11:59:38.391798       1 constructor.go:175] q68h9g3p: got 1 nodes, 32 apps in 21ms
I0630 11:59:39.607942       1 compaction.go:92] compaction iteration started
I0630 11:59:41.017718       1 constructor.go:175] 4fayh2b7: got 1 nodes, 29 apps in 50ms
I0630 11:59:42.027859       1 constructor.go:175] jh12pq95: got 1 nodes, 33 apps in 51ms
I0630 11:59:49.608405       1 compaction.go:92] compaction iteration started
I0630 11:59:49.994033       1 constructor.go:175] jh12pq95: got 1 nodes, 33 apps in 33ms
I0630 11:59:50.023657       1 constructor.go:175] 4fayh2b7: got 1 nodes, 29 apps in 29ms
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xe45448]

goroutine 13 [running]:
github.com/coroot/coroot/model.(*Flux).Merge(...)
    /tmp/src/model/flux.go:35
github.com/coroot/coroot/constructor.mergeWorlds({0x400190bbc8, 0x2, 0x4}, 0x40023158c0)
    /tmp/src/constructor/multicluster.go:40 +0x8f8
github.com/coroot/coroot/constructor.(*Constructor).LoadWorld(0x400190bd80, {0x2a3fbd0, 0x34bfb60}, 0x6a43a129, 0x6a43af39, 0xf, 0x0)
    /tmp/src/constructor/constructor.go:96 +0x474
github.com/coroot/coroot/watchers.handleProjectUpdate(0x4000056da0, 0x400045a510, 0x4000334c80, 0x40001486d8, 0x40003ad0a0, 0x400018c190, {0x40033a6fd0, 0x8})
    /tmp/src/watchers/watchers.go:172 +0x2f4
github.com/coroot/coroot/watchers.Start.func2()
    /tmp/src/watchers/watchers.go:72 +0x33c
created by github.com/coroot/coroot/watchers.Start in goroutine 1
    /tmp/src/watchers/watchers.go:56 +0x2e4
stream closed: EOF for default/coroot-coroot-0 (coroot)

```

Thanks in adavance !

--- a/constructor/multicluster.go
+++ b/constructor/multicluster.go
@@ -37,8 +37,9 @@ func mergeWorlds(worlds []*model.World, checkConfigs model.CheckConfigs) *model.
 		}
 		if w.Flux != nil {
 			if res.Flux == nil {
-				res.Flux.Merge(w.Flux)
+				res.Flux = model.NewFlux()
 			}
+			res.Flux.Merge(w.Flux)
 		}
 		if w.ArgoCD != nil {
 			if res.ArgoCD == nil {

